recipe/meta.yaml

```diff
@@ -1,12 +1,13 @@
-{% set version = "2.4.7" %}
+{% set version = "2.5.4" %}
+{% set libltdl_version = "2.4.3a" %}
```

Comment:
Where is this version coming from?

Reply:
i found it in `libltdl/configure` first line. will add a comment

Reply:
[`0883214` (#35)](https://github.com/conda-forge/libtool-feedstock/pull/35/commits/0883214c2745ae402758b6e24fda63fcf4b8e521)